### PR TITLE
[VEN-747] Edit nat spec and order vtoken contracts

### PR DESCRIPTION
**Pull request by coreyar**
## Description

Adds linting ordering and orders entities
Edits nat spec for clarity and per guidelines

### CODE REVIEW

File: contracts/Tokens/VTokens/VBNB.sol [hunk 1]
```diff
@@ -36,12 +36,22 @@ contract VBNB is VToken {
...
+        requireNoError(err, "mint failed");
+    }
+
     /*** User Interface ***/
 
     /**
      * @notice Sender supplies assets into the market and receives vTokens in exchange
      * @dev Reverts upon any failure
      */
+    // @custom:events Emits Transfer event
```
Comment (chechu) [inline contracts/Tokens/VTokens/VBNB.sol:53]: We could avoid mixing styles in the same block. I would add the `custom:events` tags into the `/** ... */` block

Comment (chechu) [reply contracts/Tokens/VTokens/VBNB.sol:53]: I'm sorry, now I read your comment in Slack. I suppose you don't want to add these comments in the final doc, right? BTW, why?

**Review by chechu — APPROVED** [review]
LGTM

**Review by defcon022 — APPROVED** [review]
LGTM

**Pull request merged**